### PR TITLE
Test-cases

### PR DESCRIPTION
**Pull request by fedeimbaguable**

### CODE REVIEW

**Review by roxanafalco20 — CHANGES_REQUESTED** [review]
Fede, I reviewed the test cases and they are okay related to content. This sprint, let's work on the visualization, because this approach is really hard to read. Can you spot the difference? Check it [here](https://drive.google.com/file/d/17UgkIy-YTFS4GjY17AX6qhDmIoRqkxzF/view?usp=sharing).

Another important aspect is that this documentation is about the test cases definition and not their execution. So, obtained result is not needed for this purpose. Related to the test cases fields, preconditions are missing. 

On the other hand, something that i’m noticing is the way that these are written. Test cases should be impersonal (in spanish or english). Is not needed to change it now, but will be really appreciated on next nano-experience. I’ll leave an example and we can continue this discussion if needed. I’m leaving [here](https://drive.google.com/file/d/1mNFkZTZKeZo8-NKBT19PWZ7zHz1XuwSt/view?usp=sharing) an example.

In summary:

- [ ] Use tables in order to show the info in a friendlier way.
- [ ] Remove “Obtained result” field from the test cases
- [ ] Add “Pre-condition/s” field to the test cases

**Review by RoxanaFalco — APPROVED** [review]
Green light on my end

**Review by roxanafalco20 — APPROVED** [review]
My bad, wrong github account.
Green light on my end :)
